RileyLinkKit/PumpOpsSynchronous.swift

```diff
@@ -15,7 +15,8 @@ public enum PumpCommsError: ErrorType {
     case RFCommsFailure(String)
     case UnknownPumpModel
     case RileyLinkTimeout
-    case UnknownResponse(String)
+    case UnknownResponse(rx: NSData?, during: String?)
+    case UnexpectedResponse(PumpMessage, from: PumpMessage)
```

Comment:
This clarifies a lot of `UnknownResponse` errors, and removes hand-crafted strings, which will make it easier to localize the user-facing portion if necessary.